pkgs/loeffel-io/ls-lint/registry.yaml

```diff
@@ -0,0 +1,54 @@
+packages:
+  - type: github_release
+    repo_owner: loeffel-io
+    repo_name: ls-lint
+    description: An extremely fast directory and filename linter - Bring some structure to your project filesystem
+    asset: ls-lint-{{.OS}}-{{.Arch}}
+    # error: opening tar archive for reading: wrapping file reader: gzip: invalid header
+    format: raw
```

Comment:
If use `format: tar.gz` an error occured by aqua.
```console
$ aqua i --test
ERRO[0001] install the package                           aqua_version=2.8.0 env=darwin/arm64 error="opening tar archive for reading: wrapping file reader: gzip: invalid header" package_name=loeffel-io/ls-lint package_version=v2.0.0 program=aqua registry=standard
```

Reply:
I'm taking a look.

https://github.com/loeffel-io/ls-lint/releases/tag/v2.0.0

The file has no problem.

```console
$ tar vtzf ls-lint-darwin-arm64.tar.gz 
-r-xr-xr-x  0 0      0        1075  1  1  2000 LICENSE
-r-xr-xr-x  0 0      0     3155906  1  1  2000 ls-lint-darwin-arm64

$ tar xvzf ls-lint-darwin-arm64.tar.gz 
x LICENSE
x ls-lint-darwin-arm64
```

I could reproduce the issue.

```console
$ aqua i
INFO[0000] create a symbolic link                        aqua_version=2.8.0 command=ls-lint env=darwin/arm64 program=aqua
INFO[0000] create a symbolic link                        aqua_version=2.8.0 command="ls-lint-{{.OS}}-{{.arch}}" env=darwin/arm64 program=aqua
INFO[0000] download and unarchive the package            aqua_version=2.8.0 env=darwin/arm64 package_name=loeffel-io/ls-lint package_version=v1.1.0 program=aqua registry=standard
INFO[0000] download and unarchive the package            aqua_version=2.8.0 env=darwin/arm64 package_name=loeffel-io/ls-lint package_version=v2.0.0 program=aqua registry=standard
INFO[0000] download and unarchive the package            aqua_version=2.8.0 env=darwin/arm64 package_name=loeffel-io/ls-lint package_version=v1.10.0-beta.1 program=aqua registry=standard
INFO[0000] download and unarchive the package            aqua_version=2.8.0 env=darwin/arm64 package_name=loeffel-io/ls-lint package_version=v1.0.0 program=aqua registry=standard
INFO[0001] downloading a checksum file                   aqua_version=2.8.0 env=darwin/arm64 package_name=loeffel-io/ls-lint package_version=v2.0.0 program=aqua registry=standard
ERRO[0001] install the package                           aqua_version=2.8.0 env=darwin/arm64 error="opening tar archive for reading: wrapping file reader: gzip: invalid header" package_name=loeffel-io/ls-lint package_version=v2.0.0 program=aqua registry=standard
FATA[0001] aqua failed                                   aqua_version=2.8.0 env=darwin/arm64 error="it failed to install some packages" program=aqua
```

```yaml
packages:
  - type: github_release
    repo_owner: loeffel-io
    repo_name: ls-lint
    description: An extremely fast directory and filename linter - Bring some structure to your project filesystem
    asset: ls-lint-{{.OS}}-{{.Arch}}.{{.Format}}
    # error: opening tar archive for reading: wrapping file reader: gzip: invalid header
    format: tar.gz
    supported_envs:
      - darwin
      - linux
      - amd64
    checksum:
      type: github_release
      asset: checksums.txt
      algorithm: sha256
    version_constraint: semver(">= 2.0.0")
    version_overrides:
      - version_constraint: semver(">= 1.10.0-beta.1")
        asset: ls-lint-{{.OS}}
        format: raw
        overrides:
          - goos: linux
            goarch: arm64
            asset: ls-lint-{{.OS}}-{{.Arch}}
          - goos: darwin
            goarch: arm64
            asset: ls-lint-{{.OS}}-{{.Arch}}
        checksum:
          enabled: false
        files:
          - name: ls-lint-{{.OS}}-{{.arch}}
      - version_constraint: semver(">= 1.1.0")
        asset: ls-lint-{{.OS}}
        format: raw
        supported_envs:
          - darwin
          - amd64
        rosetta2: true
        checksum:
          enabled: false
        files:
          - name: ls-lint-{{.OS}}-{{.arch}}
      - version_constraint: semver("< 1.1.0")
        asset: ls-lint-{{.OS}}
        format: raw
        supported_envs:
          - linux/amd64
          - darwin
        rosetta2: true
        checksum:
          enabled: false
        files:
          - name: ls-lint-{{.OS}}-{{.arch}}
```

🤔

Reply:
BTW, `aqua i`'s `--test` option was deprecated at aqua v2.0.0.

```console
$ aqua i --help | grep test
   --test                  This flag was deprecated and had no meaning from aqua v2.0.0. This flag will be removed in aqua v3.0.0. https://github.com/aquaproj/aqua/issues/1691 (default: false)
```

- https://github.com/aquaproj/aqua/issues/1691

Reply:
https://github.com/mholt/archiver/blob/cc194d2e4af2dc09a812aa0ff61adc4813ea6c69/tar.go#L158
https://github.com/mholt/archiver/blob/cc194d2e4af2dc09a812aa0ff61adc4813ea6c69/tar.go#L425
https://pkg.go.dev/compress/gzip#pkg-variables
https://pkg.go.dev/compress/gzip#NewReader
https://cs.opensource.google/go/go/+/refs/tags/go1.20.4:src/compress/gzip/gunzip.go;l=113

Reply:
> BTW, aqua i's --test option was deprecated at aqua v2.0.0.

oh! thank you for teach me!